configuration/scripts/options/set_nml.jra55

```diff
@@ -1,2 +1,3 @@
 atm_data_format = 'nc'
 atm_data_type   = 'JRA55'
+atm_data_date   = '20230919'
```

Comment:
I think this setting is incorrect.  jra55 forcing files by default will not be _20230919, only the tx1 file has that setting.